documentation-site/pages/components/breadcrumbs.mdx

```diff
@@ -27,6 +28,10 @@ the user’s location in an application.
   <BasicBreadcrumbs />
 </Example>
 
+<Example title="Breadcrumbs with pseudo separators" path="examples/breadcrumbs/pseudo.js">
```

Comment:
can you add some description here that we recommend using this for apps where SEO is important?